server/src/main/java/com/linecorp/centraldogma/server/TlsConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server;
+
+import java.io.File;
+
+import javax.annotation.Nullable;
+
+/**
+ * TLS configuration.
+ */
+public interface TlsConfig {
```

Comment:
Is there any reason you split TLS configuration into an interface and an implementation?

Reply:
At first time, they were slightly different due to the properties for specifying file path. But now I realized they became same, so I think we don't need to split them. I'll fix it. :-)